.github/workflows/cron-pins.yml

```diff
@@ -0,0 +1,35 @@
+name: Cron Pins
+
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+
+jobs:
+  update:
+    name: Update pin statuses and size
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env: ['staging', 'production']
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        uses: bahmutov/npm-install@v1
+
+      - name: Run job
+        env:
+          DEBUG: '*'
+          ENV: ${{ matrix.env }}
+          FAUNA_KEY: ${{ secrets.FAUNA_KEY }}
+          STAGING_FAUNA_KEY: ${{ secrets.STAGING_FAUNA_KEY }}
+          CLUSTER_API_URL: ${{ secrets.CLUSTER_API_URL }}
```

Comment:
this is already exposed in the wrangler.toml, so would be ok to hardcode it here.